.eslintrc.js

```diff
@@ -25,23 +25,17 @@ module.exports = {
     // node files
     {
       files: [
-        '.eslintrc.js',
-        '.template-lintrc.js',
-        'ember-cli-build.js',
-        'index.js',
-        'setup-middleware.js',
-        'cli-options-filter.js',
-        'node-tests/setup-middleware-test.js',
```

Comment:
Do we not need this anymore?

Reply:
@scalvert it should be covered by more generic `'./node-tests/**/*.js',` few lines below

Reply:
Ah sorry I missed that. Thanks!